Add hub translations

LJBabbage opened the pull request:
### What is the context of this PR?
Add the 4 hub keys to translations. To align eq-translations and runner I have had to add Gaelic. I removed Ulster Scots as it seems completely redundant, as there was a po and nothing else, however I updated the tests it had to Gaelic

### How to review 
Check the keys have been added correctly and laugh at my Welsh translations

ajmaddaford commented on doc/decisions/0001-context-in-translations.md:
```diff
@@ -18,6 +18,10 @@ For some translatable strings, we create a message context (`msgctxt`) that cont
+  | `$.hub.submission.button` | Custom hub submission button | |
+  | `$.hub.submission.guidance` | Custom hub submission guidance | |
+  | `$.hub.submission.title` | Custom hub submission title | |
+  | `$.hub.submission.warning` | Custom hub submission warning | |
```
This file doesn't need to be updated as the most up to date list is in the Python.

ajmaddaford commented on tests/schemas/cy/test_language.json:
```diff
@@ -17,6 +17,15 @@
+        "submission": {
+            "button": "Testun cyflwyno",
+            "guidance": "Canllawiau cyflwyno",
+            "title": "Arddangosir y teitl ar y canolbwynt pan fydd wedi'i gwblhau",
```
```suggestion
            "title": "Arddangosir y teitl ar y canolbwynt pan fydd wedi’i gwblhau",
```

MebinAbraham replied:
@ajmaddaford I can't quite remember what was agreed, but did we say that we would keep the `test_langauge` schemas in here and runner in sync?. If so, we should update the runner one, but obv that would break tests as is.

ajmaddaford replied:
Yes, they should be kept in sync.

LJBabbage commented:
I fixed and updated all the issue with test_languages (missing translation keys, empty values, old versions etc)for all languages only to find validator wouldn't allow me to have a final summary and a hub, so I had to add a separate schemas, doh. It seems daft not to include the changes to test_languages even thought they weren't in the end needed

MebinAbraham commented on eq_translations/schema_translation.py:
```diff
@@ -13,8 +13,8 @@ def load(self, translation_file_path):
 
-            if self.catalog.locale_identifier == "sco_ulster":
-                self.catalog.locale = Locale("eo")
+            if self.catalog.locale_identifier == "ga_IE":
+                self.catalog.locale = Locale("ga")
```
Why was this changed?

LJBabbage replied:
There was a random test_language-eo po, but no JSON to go with it, seemed completely pointless and as I had introduced a proper ga po I switched it over. It will also be easier for test schema translations if needed as Ulster Scots doesn't have one online

MebinAbraham replied:
This was added to support Ulster scots in runner. Babel doesn't support Ulster scots but does support ga_IE hence `eo` is manually set.

LJBabbage replied:
That does make sense, I weirdly thought this was in a test. I have also added a comment to clarify why we do it

ajmaddaford commented on tests/schemas/en/test_language.json:
```diff
@@ -17,13 +17,19 @@
+        "guidance": "Submission guidance",
+        "title": "Submission title",
+        "warning": "Submission warning"
+    },
```
This schema is slightly different to the one in runner - it's missing the hub enabled property and there are some minor formatting differences.

ajmaddaford commented on tests/schemas/cy/test_language.json:
```diff
@@ -17,8 +17,11 @@
+    },
     "submission": {
-        "button": "Testun cyflwyno",
+        "button": "Cnaipe aighneachta",
```
Not sure what happened here, but this looks like Gaelic rather than Welsh 😕 .

LJBabbage replied:
They are both Welsh - Testun cyflwyno = Presentation text, Cnaipe aighneachta = Submission button.
